### DISCUSSION TITLE
Daily 23/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#427] [Daily Sprint]
Con respecto al proyecto:

¿Qué hice ayer?
¿Qué haré hoy?
Impedimentos o dificultades

**Comment by rpgrca** [comment]
Ayer estuve arreglando varias cosas que reporté, además de reportar más cosas. Hoy estuve y voy a estar arreglando más cosas y reportando más cosas. A la tarde tengo clase de web así que si empiezan a hablar sobre los controladores de Django empiezo con eso también, si no intento recordar cómo eran para empezar a definir las api. Tal vez escriba en la Wiki cuáles son los entry points que vamos a usar.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve haciendo algunas tareas que tenía asignadas, pero que aún no subí al repo. 
Hoy tengo un  día bastante complicado, así que no podré hacer mucho. Quizás por la noche pueda empezar a subir algunas cosas.

**Comment by BraianTroncoso** [comment]
Ayer solamente realicé una tarea de las que iba hacer porque me sentia mal, hoy estoy un poco enfermo también, entre el trabajo y otras cuestiones. Voy asignarme un par de tareas de bugs, y ver si puedo adelantar algo.(Aunque estoy atrasado con las tareas de ayer de la tec porque no pude ver la clase de python).

**Comment by Brian-Almada** [comment]
Ayer estuve practicando Django. Hoy asistí a la clase de programación y luego revisando issues tratando de hacer alguna.
